vm/src/thread.rs

```diff
@@ -371,7 +398,34 @@ impl Thread {
     }
 
     /// Retrieves the global called `name`.
-    /// Fails if the global does not exist or it does not have the correct type.
+    ///
+    /// # Examples
+    ///
+    /// Bind the `(+)` function in gluon's prelude standard library
+    /// to an `add` function in rust
+    ///
+    /// ```rust
+    /// # extern crate gluon;
+    /// # use gluon::{new_vm,Compiler,RootedThread, Thread};
+    /// # use gluon::vm::api::{FunctionRef, Hole, OpaqueValue};
+    /// # fn main() {
+    /// let vm = new_vm();
+    /// Compiler::new()
+    ///     .run_expr_async::<OpaqueValue<&Thread, Hole>>(&vm, "example",
+    ///         r#" import! "std/prelude.glu" "#)
+    ///     .sync_or_error()
+    ///     .unwrap();
+    /// let mut add: FunctionRef<fn(i32, i32) -> i32> =
+    ///     vm.get_global("std.prelude.num_Int.(+)").unwrap();
```

Comment:
Similarily maybe we could change the example to access one of the functions on `int` instead? https://github.com/gluon-lang/gluon/blob/cc823b71d13e26570467617b3a80aa63cbf941dd/vm/src/primitives.rs#L271-L288

```
let vm = RootedThread::new();
primitives::load(&vm);
// get_global etc
```